src/date.c

```diff
@@ -363,22 +365,7 @@ static int d_parsedatestr(const char *jstr, size_t len, struct timeparts *tp,
     }
   }
 
-  /* #2: trying getdate() - it never works on some OS, but... */
-  {
-    struct tm *tm = getdate(str);
-    if (tm != NULL) {
-      tp->year = tm->tm_year + 1900;
-      tp->month = tm->tm_mon;
-      tp->day = tm->tm_mday;
-      tp->hour = tm->tm_hour;
-      tp->min = tm->tm_min;
-      tp->sec = tm->tm_sec;
-
-      return 1;
-    }
-  }
-
-  /* #3: trying toString()/toUTCString()/toDateFormat() formats */
+  /* trying toString()/toUTCString()/toDateFormat() formats */
   {
     char month[4];
     const char *frmString = " %03*s %03s %02d %d %02d:%02d:%02d %03s%d";
```

Comment:
Possible it is implementation specific?
I've checked this with clang.
%03*s skips ONLY 3 characters, while %*s skips all before whitespace.